### PR TITLE
 Added flag to allow loading of CUDA-trained parameters to CPU.

### PR DESCRIPTION
**Pull request by ztangent**
After training a model on a GPU cluster, I found myself needing to load the Pyro parameter store on a non-CUDA-enabled device. This quick patch adds a flag to make that possible using this method:
https://discuss.pytorch.org/t/on-a-cpu-device-how-to-load-checkpoint-saved-on-gpu-device/349/4

### CODE REVIEW

File: pyro/params/param_store.py [hunk 1]
```diff
@@ -170,15 +170,20 @@ def save(self, filename):
...
 
-    def load(self, filename):
+    def load(self, filename, load_to_cpu=False):
         """
         Loads parameters from disk
 
         :param filename: file name to load from
         :type name: str
+        :param load_to_cpu: flag to load CUDA-trained parameters to CPU
+        :type name: bool
```
Comment (jpchen) [inline pyro/params/param_store.py]: this is actually a typo above it should be  `:type filename: str` and `:type load_to_cpu: bool`

File: pyro/params/param_store.py [hunk 2]
```diff
@@ -170,15 +170,20 @@ def save(self, filename):
         with open(filename, "wb") as output_file:
             torch.save(self.get_state(), output_file)
 
-    def load(self, filename):
+    def load(self, filename, load_to_cpu=False):
```
Comment (fritzo) [inline pyro/params/param_store.py]: Why not simply thread through the `map_location` arg to `torch.load`? Then you could achieve what you want with
```py
pyro.load(filename, map_location=lambda s,l: 'cpu')
```
and also load to gpu via e.g.
```py
pyro.load(filename, map_location=lambda s,l: 'cuda:0')
```

**Comment by ztangent** [comment]
Made the requested changes!

**Comment by ztangent** [comment]
Wow, was extremely careless and left in the 'if load_to_cpu' line, but everything is fixed now.

**Pull request merged**